.github/workflows/base.yml

```diff
@@ -51,11 +51,21 @@ on:
       mandrel-it-issue-number:
         type: string
         description: 'The issue number to report results to mandrel-integration-tests'
+      build-stats-tag:
+        type: string
+        description: 'The tag to use for build stats upload of native tests (e.g. 22.3.0-dev-jdk17-mytest-patch-before)'
         default: "null"
+      report-build-stats:
+        type: boolean
+        description: 'Whether or not to collect and report build stats (GraalVM 22.3.0+)'
+        default: true
```

Comment:
Set to false (or actually remove the options) as I suggest in the comment in base-windows workflow